courses/free-courses-ta.md

```diff
@@ -35,6 +36,11 @@
 * [C++ Programming in Tamil](https://youtube.com/playlist?list=PLWbtDrDnmTHBPD-Dt5BJi7iP11x6UvgU0) - CS in Tamil
 
 
+### Deep Learning
+
+* [Deep Learning in Tamil](https://youtube.com/playlist?list=PLJtSFa-YIedY82pcM1RCKkqqArpQRm6tL) - Majaa Matrix
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
Not Machine Learning?

Reply:
Okay!
But it was categorized separately like that in Courses for the English language, that's why I categorized it in that manner.
So, shall I subcategorize **Machine learning** and **Deep learning** into a common category of **Artificial Intelligence**?